README.md

```diff
@@ -135,6 +135,9 @@ You can download and run pre-built versions of GATK4 from the following places:
 
 * To run a non-Spark tool, or to run a Spark tool locally, the syntax is: **`./gatk-launch ToolName toolArguments`**.
 
+* Tool arguments that allow multiple values, such as -I, can be supplied on the command line using a file with the extension “.args”. Each line of the file should contain a
```

Comment:
It looks like straight quotes are used elsewhere in this document (in both the text and in code blocks). Was it intentional to use curly quotes here?

Reply:
Thats what I get for pasting from TextEdit. Fixed.